.github/workflows/test.yml

```diff
@@ -89,12 +89,12 @@ jobs:
           doit test_examples
       - name: run coveralls
         env:
-          github-token: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
           eval "$(conda shell.bash hook)"
           conda activate test-environment
           coveralls
       - name: coveralls
```

Comment:
Maybe remov this one @kcpevey :) ?

Reply:
Yeah I had it write the first time. Thanks!